sycl/doc/GetStartedGuide.md

```diff
@@ -145,6 +150,24 @@ above.
 
 # Use DPC++ toolchain
 
+## Using the SYCL toolchain on CUDA platforms
```

Comment:
```suggestion
## Using the DPCPP toolchain on CUDA platforms
```

Reply:
DPC++ please where it's human-only readable